### PR TITLE
プログラミングで 人生の可能性を広げよう

### PR DESCRIPTION
**Pull request by Shunuk**
## What does this PR do?
Add resource

## For resources
(https://prog-8.com/)
### Description
This is online free course to learn about HTML. Also, require to make account but it is free to make. 
### Why is this valuable (or not)?
This course has a lot of section of HTML learning. And it changing the courses up to their levels. 
### How do we know it's really free?
This company has two plans of learning HTML and one of them is free. 
### For book lists, is it a book? For course lists, is it a course? etc.
 It is a course. 
## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ja.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
 ### Index
 
 * [0 - 大規模公開オンライン講座(MOOC)](#0---mooc)
+* [HTML](#HTML)
```
Comment (eshellman) [inline courses/free-courses-ja.md:4]:
```suggestion
* [HTML](#html)
```

**Review by eshellman — CHANGES_REQUESTED** [review]
also check the linter

File: courses/free-courses-ja.md (hunk 1 above)
Comment (Shunuk) [reply courses/free-courses-ja.md]: I changed it to lowercase letters. Could you check please.

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.
This resource is not a course. At best is a MOOC, but it seems only one course is free? If you really think this is a free course, please explain why. If it's a course, it should at least have a linkt to a specific course that has a matching time.

**Pull request closed without merging**